contracts/chain-adapters/L1_Adapter.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "./Base_Adapter.sol";
+import "../interfaces/AdapterInterface.sol";
+import "../interfaces/WETH9.sol";
+
+import "@uma/core/contracts/common/implementation/Lockable.sol";
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract L1_Adapter is Base_Adapter, Lockable {
+    using SafeERC20 for IERC20;
+
+    constructor(address _hubPool) Base_Adapter(_hubPool) {}
+
+    function relayMessage(address target, bytes memory message) external payable override nonReentrant onlyHubPool {
+        require(_executeCall(target, message), "execute call failed");
```

Comment:
would it be cleaner to put the `require` part in the `_executeCall` function then just call this directly? we never want the `success=false` case so we might as well assume this method does not throw if it passed. i.e the `_executeCall` would not return anything and would simply have `require(success,"execute call failed")` at the end of the function.

Reply:
Sure. Why not

Reply:
done